### PR TITLE
ensure font size h1 > h2 > h3 > h4

### PR DESCRIPTION
**Pull request by temyers**
* Settings for H2 - h4 taken from main.css
* H1 set to same size as title.

### CODE REVIEW

File: assets/style.css [hunk 1]
```diff
@@ -8,6 +8,21 @@ body {
...
+    font-size: 32px
+}
+
+h3 {
+    font-size: 26px
+}
+
+h4 {
+    font-size: 20px
+}
```
Comment (qt-pixels) [inline assets/style.css:25]: The following code snippet will:
- Setup the base font-size, line-height and margins across the website (in accordance from the 'Golden Type Scale' rule defined from (https://type-scale.com/))
- Fluid responsive font sizes (mobile first): Mobile -> Tablet -> Laptop -> Desktop -> Widescreen

`
html {
  font-size: 62.5% !important;  /*10px*/
}

body {
  font-family: 'Roboto', sans-serif;
  font-weight: 400;
  line-height: 1.5;
  font-size: 10px;
}

p, span, a, em {
  margin-bottom: 1.15rem;
  font-size: 16px;
}

h1, h2, h3, h4, h5 {
  font-weight: bold;
  line-height: 1.15;
  transition: font-size 0.2s cubic-bezier(0.4, 0, 1, 1);
}

@media screen and (min-width: 1920) {
  h1 { font-size: 5.9983em; }
  h2 { font-size: 3.75em; }
  h3 { font-size: 2.9991em; }
  h4 { font-size: 2.0243em; }
  h5 { font-size: 1.4993em; }
}

@media screen and (max-width: 1279) {
  h1 { font-size: 5.3556em; }
  h2 { font-size: 3.3333em; }
  h3 { font-size: 2.7849em; }
  h4 { font-size: 2.0243em; }
  h5 { font-size: 1.4993em; }
}

@media screen and (max-width: 959px) {
  h1 { font-size: 4.7129em; }
  h2 { font-size: 2.9167em; }
  h3 { font-size: 2.5707em; }
  h4 { font-size: 1.8219em; }
  h5 { font-size: 1.3118em; }
}

@media screen and (max-width: 599px) {
  h1 {font-size: 3.5em;}
  h2 {font-size: 2.441em;}
  h3 {font-size: 1.953em;}
  h4 {font-size: 1.563em;}
  h5 {font-size: 1.25em;}
}
`

However, after observing some Articles, it's apparent there are some custom classes which specify Heading overrides (i.e. **.post-title**). The above styles will not apply in these cases ass CSS works by specificity. 
A couple of options to solve this:
* Update all of the existing articles by removing the custom class names.
* Update the Blog article HTML generate to omit these class names.
* Updated our CSS style changes (above) with using the **! important;** notation (css-hack). 

More things to note, a lot of the blogs dont respect accessible HTML; there are articles where a blog is only using H1 tags. In reality, a page should only have 1 H1 -> couple of H2s, several H3s etc.. This **WILL** require human intervention. 

Happy to chat :)

**Comment by Hamo69** [comment]
Yo.. this going anywhere?

**Comment by temyers** [comment]
Will update CSS as per recommended.

The following is all below the line - outside thescope of this PR 
----

However, after observing some Articles, it's apparent there are some custom classes which specify Heading overrides (i.e. .post-title). The above styles will not apply in these cases ass CSS works by specificity.
A couple of options to solve this:

    Update all of the existing articles by removing the custom class names.
    Update the Blog article HTML generate to omit these class names.
    Updated our CSS style changes (above) with using the ! important; notation (css-hack).

More things to note, a lot of the blogs dont respect accessible HTML; there are articles where a blog is only using H1 tags. In reality, a page should only have 1 H1 -> couple of H2s, several H3s etc.. This WILL require human intervention.

**Comment by Hamo69** [comment]
@temyers @qt-pixels shall we close this off?

**Pull request closed without merging**